tensorflow/compiler/tf2tensorrt/convert/utils.h

```diff
@@ -79,6 +79,10 @@ using absl::StrCat;
    (NV_TENSORRT_MAJOR == major && NV_TENSORRT_MINOR == minor && \
     NV_TENSORRT_PATCH == patch && NV_TENSORRT_BUILD >= build))
 
+Status ConcatenateShape(nvinfer1::INetworkDefinition* network, string node_name,
+                        const std::vector<nvinfer1::ITensor*> size_tensors,
+                        nvinfer1::ITensor** new_shape);
```

Comment:
Returning StatusOr<nvinfer1::ITensor*> is a better way. Also, would you please add a brief comment on what the function does?

Reply:
Changed the name to ConcatenateTensors. Changed the return type t StatusOr.